workspace/workflows/jobs.py

```diff
@@ -402,6 +402,19 @@ def get_text_blocks_for_key(args) -> str:
     return json.dumps(blocks)
 
 
+def get_usage_text(args) -> str:
+    return "\n".join(
+        [
+            "Below lists the behaviour for `show [target]`. The behaviour for `show-failed [target]` is the same, but skips repos whose workflows are all successful.",
```

Comment:
Nit: "Below lists..." sounds odd to me. Suggest  just "Usage for `show [target]`"